walkthroughs/2-fuse-aggregator-and-api-management/walkthrough.adoc

```diff
@@ -16,7 +16,7 @@ Organizations modernizing their applications typically create many APIs.
 It then becomes paramount to understand the usage of these APIs and to ensure they are protected, for example, from a denial-of-service (DoS) attack.
 It is possible to protect and limit access to APIs as part of an integration.
 
-In this walkthrough, you use the *Fuse Aggregation App*, which is a demonstration app that provides data using an API.
+In this walkthrough, you use the Fuse Aggregation App, which is a demonstration app that provides data using an API.
```

Comment:
it's a proper name, need to signify that somehow

Reply:
According to the _Programs or functions_ section in chapter one of the IBM style guide, this should be normal text with the first letter of each word capitalised.  Leaving is as it is.